_data/simple-icons.json

```
@@ -1110,6 +1110,11 @@
             "hex": "A22430",
             "source": "https://www.gnu.org/graphics/social.html"
         },
+        {
+            "title": "Godot",
```

> Looks like they use "Godot Engine" for titles, and "Godot" in paragraphs. [...] I'm fine with using either.

In that case I think "Godot Engine" is more appropriate.

```suggestion
            "title": "Godot Engine",
```

---

Updated. :)